app/controllers/course/admin/discussion/topic_settings_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+class Course::Admin::Discussion::TopicSettingsController < Course::Admin::Controller
+  add_breadcrumb :edit, :course_admin_topics_path
```

Comment:
Do we need to define `en.breadcrumbs.course.admin.discussion.topic_settings.edit`?